### DISCUSSION TITLE
Guess the song 🎸

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#871] [Galaxy Trivia] [answered]
```
e|-7-----------7p0---------|---------7---8---7h8\5-|--7-5h3\2-----| 
B|-----0-----------0-------|---------0---0---------|--0-------5---| 
G|---------0-----------0---|-----0-----------------|--------------| 
D|-------------------------|-----------------------|--------------| 
A|-------------------------|-----------------------|--------------| 
E|-0-----------------------|-0---------------------|--------------|
```

**Bonus:** Name the songwriter.

**Comment by peterbaccaro** [comment, marked as answer]
The song is "**Nothing Else Matters**" and is a popular rock ballad by the band **Metallica**. It was released as a single from their fifth studio album, "Metallica" (also known as "The Black Album"), in 1992. The song was **written by James Hetfield**, the lead vocalist and rhythm guitarist of Metallica, and it is one of their most well-known and beloved songs.

"Nothing Else Matters" is known for its melodic and introspective nature, contrasting with the band's heavier and more aggressive style. It features acoustic guitar passages, heartfelt lyrics, and a memorable guitar solo. The song's lyrics touch upon themes of love, trust, and personal connection.
